implementations/rust/ockam/ockam_api/src/cli_state/nodes.rs

```diff
@@ -185,27 +189,33 @@ impl CliState {
             // Avoid killing the current process, return successfully instead.
             // This is needed when deleting nodes from another node, for example, during a reset
             if pid == process::id() {
+                debug!(name=%node_name, "node is the current process, skipping sending kill signal");
                 return Ok(());
             }
 
             // Try first with SIGTERM, if it fails, try again with SIGKILL
             let mut err = None;
             for signal in [signal::Signal::SIGTERM, signal::Signal::SIGKILL] {
```

Comment:
Could you please consider removing the loop here? I suspect it will take the same number of lines but will be so much easier to follow IMO

Reply:
done!